docs/.wordlist.txt

```diff
@@ -0,0 +1,76 @@
+artifacts
```

Comment:
`artifacts` and `behavior` are interesting; shouldn't we use `artefacts` and `behaviour` instead?

Reply:
right, that's what I mean: if we're using en_GB (which we should), shouldn't we update the uses of `artifacts` to `artefacts`, instead of allowing the `artifacts` spelling?

Reply:
But looking at other words, I suspect we use US English ( initializing, customized,...)

Reply:
we need to use `en_GB` so if the spelling check is not flagging those words, something's wrong

Reply:
If I remove it from the wordlist, the spelling check flags the word.

Reply:
i modify some RST files to use ``en_GB``